tests/ui/test/steps/organisations.steps.ts

```diff
@@ -67,6 +67,39 @@ Then('a name error is not displayed on the page', async function () {
   await organisationsPage.orgNameErrorNotVisible();
 });
 
+Then('a no results found text is displayed on organisation page', async function () {
+  await organisationsPage.orgNoResultsFound();
```

Comment:
For consistency can this step either be done in the same method as line 57 (check that the id for the error has been raised and then that correct text is on the page) or change the other steps to match this